instrumentation/github.com/emicklei/go-restful/otelrestful/internal/semconvutil/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
All the generated code should have a header describing it being generated.

```suggestion
// Code generated by "..."; DO NOT EDIT.
// Copyright The OpenTelemetry Authors
```

Reply:
I have not added it as then the file ignored by `golangci-lint`.

Reply:
SGTM. The thing I am looking for is the "do not edit" part. From the SIG meeting it sounds like this might be the key the linter ignores on. The plan from @pellared is to double check this.

Reply:
@MrAlias 

From https://github.com/golangci/golangci-lint/blob/cb6390d7ca5eea5e37d9ac92925b5ef2a6593c03/pkg/result/processors/autogenerated_exclude.go#L72-L80:

```go
// isGenerated reports whether the source file is generated code.
// Using a bit laxer rules than https://go.dev/s/generatedcode to
// match more generated code. See #48 and #72.
func isGeneratedFileByComment(doc string) bool {
	const (
		genCodeGenerated = "code generated"
		genDoNotEdit     = "do not edit"
		genAutoFile      = "autogenerated file" // easyjson
	)
```

What do you think about the following?

```
// Code created by gotmpl. DO NOT MODIFY.
// source: internal/shared/semconvutil/doc.go
```

Reply:
Done here https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3817/commits/decfa0da1449339bf6054c6891fa6c0109a529c6